spec/type_aliases/http__method_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+if Puppet::Util::Package.versioncmp(Puppet.version, '4.5.0') >= 0
```

Comment:
This shouldn't be supported according to metadata.json:
https://github.com/puppetlabs/puppetlabs-stdlib/blob/b4048fd3f5cfe365a03e8dc3e11eb2d42cefe803/metadata.json#L101

(same for the other type)

Reply:
done, i also created #1193 for the rest of them